bn-apps/memberships-management/membership-service-contracts-and-states/src/main/kotlin/net/corda/businessnetworks/membership/states/Membership.kt

```diff
@@ -135,7 +145,7 @@ enum class MembershipStatus {
 }
 
 /**
- * Simple metadata, that assigns a role with membership state
+ * Simple metadata example.
  */
 @CordaSerializable
-data class SimpleMembershipMetadata(val role : String)
\ No newline at end of file
+data class SimpleMembershipMetadata(val role : String = "", val humanReadableName : String = "")
```

Comment:
Not to be a pedant but "humanReadableName" suggests the other name is not human readable name. Which probably won't be the case in reality when people drive their own CX500 names

Reply:
We don't apologise for pedantry at R3 :)

Reply:
Hah true :) To avoid confusion - renamed to displayName